node_modules/col-core/lib/server.js

```diff
@@ -97,6 +97,24 @@ var setUpServer = module.exports.setUpServer = function() {
     return next();
   });
 
+  // Add CORS headers to support Bookmarklet requests.
+  // Cookies won't be passed in so all cross domain requests will be anonymous
+  app.use(function(req, res, next) {
```

Comment:
Okay, cool, I'll make that part of the PR that introduces Apache and the production build